orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/recover_semaphore.sls

```diff
@@ -0,0 +1,12 @@
+{%- from 'postgresql/settings.sls' import postgresql with context %}
+
+{% set configure_remote_db = salt['pillar.get']('postgres:configure_remote_db', 'None') %}
+
+{% if 'None' != configure_remote_db %}
+
+recover-services-db-remote-required:
+  cmd.run:
+    - name: echo $(date +%Y-%m-%d:%H:%M:%S) >> /var/log/recover-services-db-remote-required
```

Comment:
actually I don't get why do you need this file. Could you explain?
When would it be removed?